sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -3994,12 +3999,18 @@ def value_counts(self, subset=None, sort=False, normalize=False,
       columns = subset or list(self.columns)
 
       if dropna:
-        dropped = self.dropna()
+        # Must include subset here because otherwise we spuriously drop NAs due
+        # to columns outside our subset.
+        dropped = self.dropna(subset=subset)
```

Comment:
to confirm, this is a bug in existing implementation that got caught in a new docstring test?

Reply:
Yes. Added tear case fails in 1.5.3 as well without the fix.